### PR TITLE
Refs Ticket#407964 add unittest for forwarder chatwork service

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets

- [#407964](https://dev.sun-asterisk.com/issues/407964)

## What's this PR do ?

- [x] Add case less than into checkCondtion function
- [x] Write unittest for checkCondition

## Library
*(List package, library third party add new include version)*

- Library: google-analysis.js

## Performance

- [ ] Resolved n + 1 query
- [ ] Run explain query already
- [ ] Time run rake task : 1000 ms

## Checklist

- [x] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [x] Note purpose/reason, solution, scope of influence into ticket

## ENV note

```
```

## Notes
*(Other notes)*
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr></table>

File: tests/Unit/Services/ForwardChatworkServiceTest.php [hunk 1]
```diff
@@ -173,6 +173,39 @@ public function testCallNotFoundPayloadWithMultiCondition()
...
+        $params = ["name" => 'qtv', "age" => 20];
+        $webhook = factory(Webhook::class)->create();
+        $payload = factory(Payload::class)->create(['webhook_id' => $webhook->id]);
+        $conditionNotEqual = factory(Condition::class)->create(['payload_id' => $payload->id, 'operator' => '!=', 'field' => 'name', 'value' => 'qtv']);
+        $conditionGreater = factory(Condition::class)->create(['payload_id' => $payload->id, 'operator' => '>', 'field' => 'name', 'value' => 20]);
+        $conditionGreaterOrEqual = factory(Condition::class)->create(['payload_id' => $payload->id, 'operator' => '>=', 'field' => 'name', 'value' => 20]);
+        $conditionLess = factory(Condition::class)->create(['payload_id' => $payload->id, 'operator' => '<', 'field' => 'name', 'value' => 10]);
+        $conditionLessOrEqual = factory(Condition::class)->create(['payload_id' => $payload->id, 'operator' => '<=', 'field' => 'name', 'value' => 10]);
+        $conditionNotExist = factory(Condition::class)->create(['payload_id' => $payload->id, 'operator' => '<>', 'field' => 'name', 'value' => 10]);
+
```
Comment (ghost) [inline tests/Unit/Services/ForwardChatworkServiceTest.php:188]: thiếu thằng equal rồi

**Review by ghost — APPROVED** [review]

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr></table>

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr></table>

**Review by nguyenthinhhn — APPROVED** [review]

**Comment by phucqd-1305** [comment]
Ok

**Pull request merged**